src/ontology/mondo-ingest-odk.yaml

```diff
@@ -33,6 +33,8 @@ components:
       source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
     - filename: icd10who.owl
       source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
+    - filename: icd11.owl
+      source: https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
```

Comment:
## `.gz` problem?
<details><summary>Details</summary>
<p>

### Overview
I haven't looked fully into this yet, but I'll check to make sure that the fact that this is the only `source` which needs to be decompressed first doesn't cause any problems.

Obviously I'll have to decompress it, but I think the problem here is that the `Makefile` (a file that is dynamically created and that should not edit) will assume it's going to be working with a decompressed file. See this example for ICD10WHO:

```make
.PHONY: component-download-icd10who.owl
component-download-icd10who.owl: | $(TMPDIR)
	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl \
	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
```

### Possible solutions
a. **ODK goal update**: Update ODK so that when we run `update_repo` and the `Makefile` is created / updated, it has an additional step in the `component-download-SOURCE` goal where it would first detect a file that needs to be decompressed first and does so.
b. **Create our own download URL**: I would re-open the `icd11` repo, and the sole purpose of this repo would be to have a goal that would take the `.gz`, decompress, and upload a release.

</p>
</details>

Reply:
My first thought is to add a goal to download the file and unzip it.

Reply:
Sure ROBOT is not able to decompress? https://robot.obolibrary.org/convert

Reply:
@twhetzel That works best with _solution b_ in _details_ above. Or it could be done via _solution a_, but that requires an ODK PR. Looks like that's moot though because of what Nico just mentioned.

@matentzn Thanks, I think I saw something about this in the past but forgot. That's awesome that all robot commands can accept `.gz` as input or output.